src/main/java/ua/lviv/iot/parkingServer/utils/DateToday.kt

```diff
@@ -0,0 +1,10 @@
+package ua.lviv.iot.parkingServer.utils
+
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.Calendar
+
+object Util {
+        private val DATE_FORMAT: DateFormat = SimpleDateFormat("yyyy-MM-dd")
+        fun getDateTodayInString(): String = DATE_FORMAT.format(Calendar.getInstance().time)
```

Comment:
Sorry, sometimes I write a code in discussions that bad-formatted. 

Please, use shortcut `cmnd+option+L`:
```kotlin
object Util {
    |<--- 4 spaces
    private val DATE_FORMAT: DateFormat = SimpleDateFormat("yyyy-MM-dd")

```